readme.md

```diff
@@ -288,6 +288,7 @@ Thanks for contributing! 🦋🙌
 - [](# "same-branch-author-commits") [Preserves current branch and path when viewing all commits by an author.](https://user-images.githubusercontent.com/44045911/148764372-ee443213-e61a-4227-9219-0ee54ed832e8.png)
 - [](# "easy-toggle-commit-messages") [Enables toggling commit messages by clicking on the commit box.](https://user-images.githubusercontent.com/1402241/152121837-ca13bf8a-9b7f-4517-8e8d-b58bb135523b.gif)
 - [](# "link-to-compare-diff") [Linkifies the "X files changed" text on compare pages to allow jumping to the diff.](https://user-images.githubusercontent.com/46634000/157072587-0335357a-18c7-44c4-ae6e-237080fb36b4.png)
+- [](# "conventional-commits") [Displays conventional commit types as labels before the commit message.](https://user-images.githubusercontent.com/assets/2a5ae00e-f440-49ef-b810-aa9a5a2ac269.png)
```

Comment:
I think you updated the snapshot but left this broken link in here. Make sure to use the right link here. 

Also our screenshot are always with the light theme, can you take another one?

Reply:
This is what it looks in light mode: ![image](https://github.com/user-attachments/assets/739264d7-b4b2-425d-91bb-d91c42c1ab21)